src/app/scam/components/templates/surveys/surveys.component.html

```diff
@@ -0,0 +1,58 @@
+<div *ngIf="!survey" class="p2">
+  <button (click)="createSurvey()" mat-raised-button color="accent">Create survey</button>
+</div>
+
+<div *ngIf="survey" class="flex">
+  <div class="editor p2">
+    <h4>Editor</h4>
+    <div [formGroup]="formDetails" class="flex flex-column">
+      <mat-form-field>
+        <mat-label>Label</mat-label>
+        <input matInput type="text" formControlName="Label">
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>Hint</mat-label>
+        <input matInput type="text" formControlName="Hint">
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>Variable name</mat-label>
+        <input matInput type="text" formControlName="VariableName">
+      </mat-form-field>
+      <mat-form-field>
+        <mat-label>Type</mat-label>
+        <mat-select formControlName="Type">
+          <mat-option *ngFor="let type of types" [value]="type">
+            {{type}}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
```

Comment:
When the user selects input of type "select" an additional textarea for select options should appear below. 
With `formControlName="Options"`